wdl-engine/src/eval.rs

```diff
@@ -370,3 +388,280 @@ impl EvaluatedTask {
         Ok(())
     }
 }
+
+/// Represents mount points for mapping host and guest paths for task execution
+/// backends that use containers.
+#[derive(Debug, Default)]
+pub struct MountPoints(Vec<(PathBuf, PathBuf, bool)>);
```

Comment:
Can you add to this doc comment what the contents refer to? I can see from `insert` it's `(host_path, guest_path, read_only)` but would be good to go in this comment